### PR TITLE
Feature/new helper tags

### PR DESCRIPTION
**Pull request by ZanzyTHEbar**
###  I have created custom nodms, solder, and Wokwi json files to teach people: 

1. How to properly solder - with examples.

2. Not to dm people without their consent. 

3. Wokwi button - to link people to Wokwi simulator

### CODE REVIEW

**Comment by ZanzyTHEbar** [comment]
no reviews yet - How active is this?

**Comment by hox** [comment]
Hi! We're discussing this internally and will review this when we can.

**Comment by ZanzyTHEbar** [comment]
eh! A response - cool. All i needed ;) Thanks, wasn't sure if this github was still being actively maintained.

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,10 @@
+{
```
Comment (max-bromberg) [inline .vscode/settings.json]: VSCode configuration files are not necessary. This file can be removed.

File: src/tags/nodms.json [hunk 2]
```diff
@@ -0,0 +1,35 @@
+{
```
Comment (max-bromberg) [inline src/tags/nodms.json]: After discussing this tag internally we have determined we will not be adding tags that repost rules available in our code of conduct channel. This file can be removed.

File: src/tags/solder.json [hunk 3]
```diff
@@ -0,0 +1,60 @@
+{
```
Comment (max-bromberg) [inline src/tags/solder.json:1]: We would like to add this soldering tag! However, the button and its reply are not needed.

File: src/tags/wokwi.json [hunk 4]
```diff
@@ -0,0 +1,36 @@
+{
```
Comment (max-bromberg) [inline src/tags/wokwi.json]: As of now, we will not be linking to particular third-party sites through tags in this way. This file can be removed.

**Review by max-bromberg — CHANGES_REQUESTED** [review]
Please read individual file comments.
TLDR; Please remove settings.json, nodms.json, wokwi.json + edit solder.json

Thank you for your contribution! If you have any questions feel free to ask. Once these changes are made I will merge this request.

File: .vscode/settings.json (hunk 1 above)
Comment (ZanzyTHEbar) [reply .vscode/settings.json]: Yeah - this makes sense. Didn't really mean to upload it anyhow - just something i use for my personal development. Changes have been made.

File: src/tags/nodms.json (hunk 2 above)
Comment (ZanzyTHEbar) [reply src/tags/nodms.json]: Dang, I understand, but it's sad. I am so tired of having to tell people this and link them to the #welcome channel - they nevecr read it and people even argue with me that since it's not in rules they don't have to follow it - or say that i am being rude for telling them not to dm me - i really feel like this needs to be AT LEAST moved to the #code-of-conduct section. However, that's jsut a pesonal greivance - thank you for considering my changes.

File: src/tags/solder.json (hunk 3 above)
Comment (ZanzyTHEbar) [reply src/tags/solder.json:1]: Understood, i have resolved for the requested changes.

**Review by max-bromberg — APPROVED** [review]
Looks good, thanks for making those changes!
